<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>980</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="verticalSpacing">
      <number>8</number>
     </property>
     <item row="11" column="0" colspan="2">
      <widget class="QPushButton" name="pushButtonSave">
       <property name="text">
        <string>Сохранить в файл</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="pushButtonAdd">
       <property name="text">
        <string>Добавить вопрос (Task) в набор 1</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QSpinBox" name="spinBoxGet"/>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Результат операций над наборами 1 и 2:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pushButtonDel">
       <property name="text">
        <string>Удалить вопрос по номеру из набора 1</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QSpinBox" name="spinBoxDel_2"/>
     </item>
     <item row="5" column="4">
      <widget class="QPushButton" name="pushButtonGet_2">
       <property name="text">
        <string>Получить вопрос по номеру из набора 2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QSpinBox" name="spinBoxDel"/>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="pushButtonGet">
       <property name="text">
        <string>Получить вопрос по номеру из набора 1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>217</red>
             <green>217</green>
             <blue>217</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>217</red>
             <green>217</green>
             <blue>217</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>240</red>
             <green>240</green>
             <blue>240</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextSelectableByMouse</set>
       </property>
       <property name="backgroundVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="3" colspan="2">
      <widget class="QPlainTextEdit" name="plainTextEdit_2">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>217</red>
             <green>217</green>
             <blue>217</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>217</red>
             <green>217</green>
             <blue>217</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>240</red>
             <green>240</green>
             <blue>240</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="3" colspan="2">
      <widget class="QPushButton" name="pushButtonOpen_2">
       <property name="text">
        <string>Загрузить из файла</string>
       </property>
      </widget>
     </item>
     <item row="11" column="3" colspan="2">
      <widget class="QPushButton" name="pushButtonSave_2">
       <property name="text">
        <string>Сохранить в файл</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5" rowspan="14">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QSpinBox" name="spinBoxGet_2"/>
     </item>
     <item row="0" column="3" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Набор 2:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QSpinBox" name="spinBoxGenerateTest_2"/>
     </item>
     <item row="3" column="3" colspan="2">
      <widget class="QPushButton" name="pushButtonAdd_2">
       <property name="text">
        <string>Добавить вопрос (Task) в набор 2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Набор 1:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="pushButtonOpen">
       <property name="text">
        <string>Загрузить из файла</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="pushButtonGenerateTest">
       <property name="text">
        <string>Генерация набора Test объемом не более K вопросов</string>
       </property>
      </widget>
     </item>
     <item row="11" column="6">
      <widget class="QPushButton" name="pushButtonSave_3">
       <property name="text">
        <string>Сохранить в файл</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="pushButtonDel_2">
       <property name="text">
        <string>Удалить вопрос по номеру из набора 2</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QSpinBox" name="spinBoxGenerateTest"/>
     </item>
     <item row="6" column="4">
      <widget class="QPushButton" name="pushButtonGenerateTest_2">
       <property name="text">
        <string>Генерация набора Test объемом не более K вопросов</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6" rowspan="10">
      <widget class="QPlainTextEdit" name="plainTextEdit_3">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>217</red>
             <green>217</green>
             <blue>217</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>217</red>
             <green>217</green>
             <blue>217</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>240</red>
             <green>240</green>
             <blue>240</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="3" colspan="2">
      <widget class="QPushButton" name="pushButtonMerge">
       <property name="text">
        <string>-&gt; Слияние 2х наборов (все вопросы из наборов)</string>
       </property>
      </widget>
     </item>
     <item row="9" column="3" colspan="2">
      <widget class="QPushButton" name="pushButtonIntersection">
       <property name="text">
        <string>-&gt; Пересечение 2x наборов (совпадающие вопросы)</string>
       </property>
      </widget>
     </item>
     <item row="10" column="3" colspan="2">
      <widget class="QPushButton" name="pushButtonDiff">
       <property name="text">
        <string>-&gt; Разность 2х наборов (вычитаем из первого второй)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
